### ISSUE TITLE
Question: reason for chalk() approach?

### ISSUE THREAD

**Issue by elliot-nelson** [sywac/sywac-style-basic#12]
I've been working on a couple styling ideas for alternate styles for sywac, and I was curious about your approach to chalk.

Is that approach purely driven by the way you wrote the unit tests? Or was the intent that the CLI program could reach in and modify your copy of chalk if desired, e.g.:

```js
const styleBasic = require('sywac-style-basic');
styleBasic.chalk.level = 0;
```

If it's the latter I wonder if it would make sense to make it more of an injection, where for example, either of the following would work:

```js
const styleBasic = require('sywac-style-basic');
const chalk = require('chalk');

// valid
sywac.style(styleBasic);

// also valid
sywac.style(styleBasic(chalk));
```

**Comment by nexdrew** [comment]
I think the only reasoning I had was:

1. I want it to be lazily loaded, so that we're not requiring `chalk` unless it's actually needed (i.e. if help text is actually generated)
2. Since this package has a dependency on `chalk`, I figured it would be nice to expose it so the package using `sywac-style-basic` doesn't have to have its own dependency on `chalk` if it wants to use it for whatever reason

Number 1 is the important part here. Number 2 is just a theoretical convenience.

I don't think I considered injecting or manipulating `chalk` as an intended use-case when I wrote this package.
